src/styles/devices.styl

```diff
@@ -0,0 +1,15 @@
+@import 'cozy-ui'
+
+.devices-view
+    h2
```

Comment:
I think h2 from the account view and the device view should share the same style, and a common class, but there's some serious shared style to do here. But maybe not for now.